src/index.ts

```diff
@@ -256,6 +257,19 @@ export const replacer = function replacer(options: Options): any {
         return `_duplicate_${JSON.stringify(keys)}`;
       }
 
+      if (value instanceof Error && options.allowError) {
+        return {
+          ...value,
+          // @ts-expect-error cause is not defined in the current tsconfig target(es2020)
+          ...(value.cause ? { cause: value.cause } : {}),
+          isConvertedError: true,
```

Comment:
Maybe something like this?

```suggestion
          __isConvertedError: true, 
```

Reply:
```js
{
  originalValues: ...
  isConvertedError: true, 
  constructorName: blah
}
```

Reply:
thanks for the idea, I changed it!